GVRf/Framework/src/org/gearvrf/GVRContext.java

```diff
@@ -341,6 +341,10 @@ public GVRMesh createQuad(float width, float height) {
                 width * 0.5f, height * -0.5f, 0.0f };
         mesh.setVertices(vertices);
 
+        float[] normals = { 0.0f, 0.0f, 1.0f, 0.0f, 0.0f, 1.0f, 0.0f, 0.0f,
+                1.0f, 0.0f, 0.0f, 1.0f };
+        mesh.setNormals(normals);
+
         float[] texCoords = { 0.0f, 0.0f, 0.0f, 1.0f, 1.0f, 0.0f, 1.0f, 1.0f };
         mesh.setTexCoords(texCoords);
```

Comment:
Just my personal opinion, since these normals and texCoords are constant values. I think we'd better to make them as final static member of this class.